ros/src/twist_controller/twist_controller.py

```diff
@@ -54,7 +54,7 @@ def control(self, *args, **kwargs):
         else:
             vel = current_vel
 
-        steering = self.yaw_controller.get_steering(linear_vel, angular_vel, vel)
```

Comment:
rename the variable from argular_vel to angular_vel, this is just a typo in the variable name..